### PR TITLE
[4.0] Fix select menu item 500

### PR DESCRIPTION
**Pull request by infograf768**
### Summary of Changes
Missing correction in #32208

### Testing Instructions
Create a new Menu item and click on Select button

### Actual result BEFORE applying this Pull Request
Screen Shot 2021-01-31 at 09 45 47

### Expected result AFTER applying this Pull Request
Screen Shot 2021-01-31 at 10 15 00

@dgrammatiko

### CODE REVIEW

**Comment by Fedik** [comment]
I have tested this item :white_check_mark: successfully on f3b2b2c4128753663f6fcbe245ef9695027bfdfa<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32218">issues.joomla.org/tracker/joomla-cms/32218</a>.</sub>

File: libraries/src/HTML/Helpers/Bootstrap.php [hunk 1]
```diff
@@ -581,7 +581,10 @@ public static function startAccordion($selector = '.myAccordian', $options = [])
 		}
 
 		// Include Bootstrap component
-		HTMLHelper::_('bootstrap.loadComponent', 'collapse');
+		Factory::getApplication()
```
Comment (dgrammatiko) [inline libraries/src/HTML/Helpers/Bootstrap.php:584]: `HTMLHelper::_('bootstrap.collapse', $selector);` should be also working here

Comment (Fedik) [reply libraries/src/HTML/Helpers/Bootstrap.php:584]: they will have differrent options set, `bootstrap.accordion` options vs `bootstrap.collapse` options.
it better as @infograf768 made

Comment (dgrammatiko) [reply libraries/src/HTML/Helpers/Bootstrap.php:584]: Can pass the $opt as 3rd param and remove the addScriptOptions from here

**Comment by dgrammatiko** [comment]
I have tested this item :white_check_mark: successfully on f3b2b2c4128753663f6fcbe245ef9695027bfdfa<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32218">issues.joomla.org/tracker/joomla-cms/32218</a>.</sub>

**Comment by infograf768** [comment]
RTC. Thanks for testing<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32218">issues.joomla.org/tracker/joomla-cms/32218</a>.</sub>

**Pull request merged**